package.json

```
@@ -1,14 +1,14 @@
 {
-    "name": "minecraft-java-core",
-    "version": "3.10.4",
+    "name": "minecraft-java-core-azbetter",
+    "version": "3.11.0",
     "types": "./build/Index.d.ts",
     "exports": {
         ".": {
             "import": "./build/Index.js",
             "require": "./build/Index.js"
         }
     },
-    "description": "A library starting minecraft game NW.js and Electron.js",
+    "description": "A library starting minecraft game NW.js and Electron.js with better Azuriom support.",
```

Again, you'll probably want to remove this part
```suggestion
    "description": "A library starting minecraft game NW.js and Electron.js",
```